src/AppInstallerRepositoryCore/InstallerMetadataCollectionContext.cpp

```diff
@@ -0,0 +1,900 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "winget/InstallerMetadataCollectionContext.h"
+
+#include <AppInstallerDownloader.h>
+#include <AppInstallerErrors.h>
+#include <AppInstallerLogging.h>
+#include <AppInstallerStrings.h>
+
+#include <winget/ManifestJSONParser.h>
+
+using namespace AppInstaller::Utility;
+
+namespace AppInstaller::Repository::Metadata
+{
+    namespace
+    {
+        struct ProductMetadataFields_1_0
+        {
+            utility::string_t ProductVersionMin = L"productVersionMin";
+            utility::string_t ProductVersionMax = L"productVersionMax";
+            utility::string_t Metadata = L"metadata";
+            utility::string_t InstallerHash = L"installerHash";
+            utility::string_t SubmissionIdentifier = L"submissionIdentifier";
+            utility::string_t Version = L"version";
+            utility::string_t AppsAndFeaturesEntries = L"AppsAndFeaturesEntries";

```

Comment:
>A [](http://example.com/codeflow?start=57&length=1)

Is it intentional to have some values start with lower case and some values start with upper case?

Reply:
It is in that those that start with upper case are the same names as from the manifest JSON schema.